packages/typegpu/src/std/numeric.ts

```diff
@@ -264,15 +268,14 @@ function cpuCountLeadingZeros<T extends AnyIntegerVecInstance>(value: T): T;
 function cpuCountLeadingZeros<T extends AnyIntegerVecInstance | number>(
   value: T,
 ): T {
-  throw new Error(
-    'CPU implementation for countLeadingZeros not implemented yet. Please submit an issue at https://github.com/software-mansion/TypeGPU/issues',
-  );
+  throw new MissingCpuImplError('countLeadingZeros');
```

Comment:
When I first looked at this, I thought that's the message was going to be shown to the user, but now I can see `cpuCountLeadingZeros` is only being used for types. Maybe we should make this more explicit to the devs seeing the code?

```suggestion
  throw new Error('Unreachable, only used as types');
```